docs/basic-usage/teams-permissions.md

```diff
@@ -96,3 +94,25 @@ class YourTeamModel extends \Illuminate\Database\Eloquent\Model
     // ...
 }
 ```
+
+## Optional Trait for Teams
+
+`TeamHasRoles` trait must be added to the Team model to enable features like cascade deleting.
+
+Thus, a typical basic Team model would have these basic minimum requirements:
+
+```php
+use Spatie\Permission\Traits\TeamHasRoles;
+
+class YourTeamModel extends \Illuminate\Database\Eloquent\Model
+{
+    use TeamHasRoles;
+
+    // ...
+}
+```
+
+**Prerequisites for `TeamHasRoles`**
+- Team model must not have a `permission` or `permissions` property, nor a `permissions()` method
+- Team model must not have a `role` or `roles` property, nor a `roles()` method
+- Team model must not have a `especific_role` or `especific_roles` property, nor a `especific_roles()` method
```

Comment:
@erikn69 was the `e` prefix to `especific_roles` intentional?

Reply:
No, sorry, i'm still learning english
But  I think this PR could be unnecessary, maybe as a guide

Reply:
I will fix it